test/unit_test/deepdoc/parser/test_monkeyocrv2_parser.py

```
@@ -0,0 +1,96 @@
+def make_zip(name, text, *, label="Text", include_markdown=True):
+    out = io.BytesIO()
+    with zipfile.ZipFile(out, "w") as z:
+        if include_markdown:
+            z.writestr(f"{name}/{name}.md", text)
+        z.writestr(f"{name}/jsons/{name}.json", json.dumps({"layouts": [{"label": label, "content": text, "bbox": [1, 2, 30, 40], "page_num": 2}]}))
+    return out.getvalue()
+
+
+def test_converts_native_layout_to_sections(monkeypatch):
+    class Response:
+        headers = {"content-type": "application/zip"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)